### PR TITLE
docs/unified/novacustom/camera-switch.md: added documentation

### PR DESCRIPTION
**Pull request by Mixss**

### CODE REVIEW

File: docs/unified/novacustom/camera-switch.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Camera switch setting
+
+Novacustom laptops have the option to disable the webcam completely. After
+disabling this option, the power to the camera is completely cut off.
+
```
Comment (mkopec) [inline docs/unified/novacustom/camera-switch.md:5]: This section could use a photo or a screenshot of the feature in the setup menu

File: docs/unified/novacustom/camera-switch.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Camera switch setting
+
+Novacustom laptops have the option to disable the webcam completely. After
```
Comment (mkopec) [inline docs/unified/novacustom/camera-switch.md:3]:
```suggestion
NovaCustom laptops have the option to disable the webcam completely. After
```

File: docs/unified/novacustom/camera-switch.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+disabling this option, the power to the camera is completely cut off.
+
+## Changing setting
+
+You can find **Enable Camera** option in **Dasharo System Features** in submenu
+**Dasharo Security Option**. By default the option is enabled.
```
Comment (mkopec) [inline docs/unified/novacustom/camera-switch.md:9]:
```suggestion
**Dasharo Security Options**. By default the option is enabled.
```

**Review by mkopec — CHANGES_REQUESTED** [review]

File: docs/unified/novacustom/camera-switch.md (hunk 1 above)
Comment (Mixss) [reply docs/unified/novacustom/camera-switch.md:5]: added in 42563eee50bae817d0b20418e1b73d34775acafc

File: docs/unified/novacustom/camera-switch.md (hunk 2 above)
Comment (Mixss) [reply docs/unified/novacustom/camera-switch.md]: fixed in 42563eee50bae817d0b20418e1b73d34775acafc

File: docs/unified/novacustom/camera-switch.md (hunk 3 above)
Comment (Mixss) [reply docs/unified/novacustom/camera-switch.md]: fixed in 42563eee50bae817d0b20418e1b73d34775acafc

**Review by mkopec — APPROVED** [review]

**Pull request merged**